template.yaml

```diff
@@ -31,6 +31,14 @@ Parameters:
     Type: String
     Default: EDGE
     Description: The ApiGateway Endpoint Type
+  SearchInfrastructureApiUri:
+    Type: String
+    Description: URI to external search infrastructure API (SWS).
+    Default: 'https://api.sws.aws.sikt.no'
+  SearchInfrastructureAuthUri:
+    Type: String
+    Description: URI to auth for external search infrastructure API (SWS).
+    Default: 'https://sws-auth.auth.eu-west-1.amazoncognito.com/oauth2/token'
```

Comment:
These parameters should be also set in the NVA-infrastucture if there are multiple possible values to them